projects/habitat_ovmm/configs/agent/rl_agent.yaml

```diff
@@ -34,12 +34,13 @@ SKILLS:
     checkpoint_path: data/checkpoints/ovmm_baseline_home_robot_challenge_2023/gaze_at_obj.pth
     rl_config: projects/habitat_ovmm/configs/agent/skills/gaze_rl.yaml # with continuous actions
     gym_obs_keys:
-      - robot_head_depth
+      - head_depth
```

Comment:
Will this break things for anyone?

Reply:
Yes, this will cause an error if someone tries to load the policies they trained in the previous version. But those policies are anyways not expected to work due to difference in physics simulation.

Reply:
I think this is fine